msteams-platform/toolkit/use-CICD-template.md

```diff
@@ -46,8 +46,17 @@ You can create the following workflow templates with GitHub:
 **Teams Toolkit in Visual Studio Code**
 
 1. Create a new Teams app project using Teams Toolkit.
-1. Select **Teams Toolkit** icon :::image type="icon" source="../assets/images/teams-toolkit-v2/add-API/api-add-icon.png" alt-text="api icon"::: from the left navigation bar.
+
+1. Select **Teams Toolkit** icon :::image type="icon" source="../assets/images/teams-toolkit-v2/add-API/api-add-icon.png"::: from the left navigation bar.
```

Comment:
left pane. remove "navigation". We don't use navigation as per MSTP standards

Reply:
changes are done